src/test/java/org/elasticsearch/index/query/TemplateQueryTest.java

```diff
@@ -371,6 +376,45 @@ public void testIndexedTemplate() throws Exception {
         assertHitCount(sr, 4);
     }
 
+    // Relates to #10397
+    @Test
+    public void testIndexedTemplateOverwrite() throws Exception {
+        createIndex("testindex");
+        ensureGreen("testindex");
+
+        index("testindex", "test", "1", jsonBuilder().startObject().field("searchtext", "dev1").endObject());
+        refresh();
+
+        PutIndexedScriptResponse scriptResponse = client().preparePutIndexedScript(MustacheScriptEngineService.NAME, "git01", 
```

Comment:
I must be missing something: I don't find a client().preparePutTemplate(...) nor client().putTemplate(...)

Can you elaborate please?